gallery/src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import packageJson from '../../package.json';
+
+const plaidVillain = require('./plaid-villain.svg');
+const tartanLogotype = require('./tartan-logotype.svg');
+
+import './PageHeader.scss';
+
+const PageHeader = () => (
+  <header className="nx-gallery-page-header">
+    <div className="nx-gallery-page-header__inner">
+      <h1 className="nx-gallery-page-header__logo">
+        <a className="nx-gallery-page-header__home-link"
+           href="#"
+           aria-label="home">
```

Comment:
This prevents a screenreader from reading any of the content within this element.  It just reads the whole thing as "home".

Reply:
> This prevents a screenreader from reading any of the content within this element. It just reads the whole thing as "home".

Removed it here: c400966dcd9ada6ecae2cc9d764dcb3363e9fe22